app/Models/FilmInfo.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class FilmInfo extends Model
+{
+    use HasFactory;
+    protected $table='film_info';
+    protected $fillable = array('name', 'film_id','duration','actors','screenwriter');
```

Comment:
in your table you don't have column `name`, also check if all columns are present in fillable